assembly/assembly-main/pom.xml

```diff
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>org.eclipse.che</groupId>
             <artifactId>assembly-wsagent-server</artifactId>
-            <type>zip</type>
+            <type>tar.gz</type>
```

Comment:
it can be an issue for other places and tools that is configured to use zip. Maybe we should generate both?
BTW what is the reason to change zip to tar.gz?